cstool/cstool.c

```diff
@@ -66,7 +66,8 @@ static struct {
 	{ "hd6309", CS_ARCH_M680X, CS_MODE_M680X_6309 },
 	{ "hcs08", CS_ARCH_M680X, CS_MODE_M680X_HCS08 },
 	{ "evm", CS_ARCH_EVM, 0 },
-	{ "riscv", CS_ARCH_RISCV, CS_MODE_LITTLE_ENDIAN },
+	{ "riscv", CS_ARCH_RISCV, CS_MODE_RISCV32 },
+	{ "riscv", CS_ARCH_RISCV, CS_MODE_RISCV64 },
```

Comment:
no you cannot repeat these as "riscv" & "riscv", because they are used as the arch+mode param for `cstool`. so they cannot be the same. perhaps you can change that to "riscv32" & "riscv64".